regression/Makefile

```diff
@@ -2,35 +2,36 @@
 # running tests in parallel, it is important that these directories are
 # listed with decreasing runtimes (i.e. longest running at the top)
 DIRS = cbmc \
-       cbmc-library \
```

Comment:
:question: I _assume_ this is just a reordering of the Makefile? If it is, it'd be best pulled out into its own commit so can be clearly stated as such